Lib/defcon/objects/layoutEngine.py

```diff
@@ -38,15 +38,15 @@ def _layoutEngineOTLTablesRepresentationFactory(layoutEngine):
             addOpenTypeFeatures(feaPath, otf)
         except:
             import traceback
-            print traceback.format_exc(5)
+            print(traceback.format_exc(5))
         finally:
-            # XXX clean up file on disk
+            os.close(fd)
             os.remove(feaPath)
-        if otf.has_key("GDEF"):
```

Comment:
It seems to be the case since 2003, see: https://github.com/behdad/fonttools/commit/8df8f6385e045bc70f538a0d047ea6d13c9cacf0#diff-6f21ae791a655cd2988257794a5ae946R293